src/port/devcons.rs

```diff
@@ -1,67 +0,0 @@
-// Racy to start.
```

Comment:
In general, I'd have preferred that this was from a file that was `git mv`'ed or `git cp`'ed (it makes it easier to see the actual diffs) but this is small enough that it's not worth quibbling about.